src/azure-cli/azure/cli/command_modules/acs/_help.py

```diff
@@ -737,6 +737,9 @@
   - name: --customer-admin-group-id
     type: string
     short-summary: The Object ID of an Azure Active Directory Group that memberships will get synced into the OpenShift group "osa-customer-admins". If not specified, no cluster admin access will be granted.
+  - name: --workspace-resource-id
```

Comment:
would something like `--loganalytics-workspace-resource-id` be better here?  I know it's a lot to type.  Don't mind whether it's loganalytics or monitoring or some other name.

Reply:
To reduce complexity, I'm okay with the long `--loganalytics-workspace-resource-id` name.  I can change it to that to provide a concise naming convention.